localize/comments/15/NuGet.Core.resources.dll.lci

```diff
@@ -1,459 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
do you know where we are using this file? looks like we don't localize exe by using mircoBuild, nuget.exe itself contains all translation files in its resource, I'm thinking this file was used by VSIX when we referenced nuget.core in VSIX.